drivers/pwm/pwm_stm32.c

```diff
@@ -830,22 +865,34 @@ static void pwm_stm32_irq_config_func_##index(const struct device *dev)        \
 }
 #define CAPTURE_INIT(index)                                                                        \
 	.irq_config_func = pwm_stm32_irq_config_func_##index,                                      \
-	.four_channel_capture_support = DT_INST_PROP(index, four_channel_capture_support)
+	.four_channel_capture_support = DT_INST_PROP(index, four_channel_capture_support),
 #else
 #define IRQ_CONFIG_FUNC(index)
 #define CAPTURE_INIT(index)
 #endif /* CONFIG_PWM_CAPTURE */
 
+#define DEFINE_TIMINGS(index)						\
+	static const uint32_t pwm_channel_timings_##index[] =		\
+		DT_INST_PROP_OR(index, timings, {});
+#define USE_TIMINGS(index)								   \
```

Comment:
An empty new line before this line makes it more beautiful :)

Reply:
Done